src/oatpp/web/server/HttpConnectionHandler.hpp

```diff
@@ -37,6 +37,7 @@ namespace oatpp { namespace web { namespace server {
 class HttpConnectionHandler : public base::Countable, public network::ConnectionHandler {
 private:
   std::shared_ptr<HttpProcessor::Components> m_components;
+  std::shared_ptr<std::atomic_ulong> m_spawns;
```

Comment:
Let's have it `signed` to avoid rounding in case of an error.
It also makes sense to add a redundant check after decrementing this value that it didn't go less than 0.

Reply:
I couldn't possibly think of a scenario where this gets more decremented than incremented. Do you have a hint?

Reply:
> I couldn't possibly think of a scenario where this gets more decremented than incremented. Do you have a hint?

Except of the missing copy-constructor ;D